src/terminal/kitty.rs

```diff
@@ -0,0 +1,256 @@
+// Copyright 2018-2019 Sebastian Wiesner <sebastian@swsnr.de>
```

Comment:
You may want to add yourself here; it's your code after all :)

Reply:
Yes, indeed :)